src/test/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryLocationLiveTest.java

```diff
@@ -236,12 +207,52 @@ public void testCreateInstanceOfNonSupportedPlan() {
         cloudFoundryPaasLocation.createServiceInstance(params.getAllConfig());
     }
 
+    @Test(groups = {"Live"})
+    public void testBindServiceToApplication() {
+        ConfigBag params = getDefaultApplicationConfiguration();
+        createApplicationStartAndCheck(params.getAllConfig());
+        createServiceAndCheck(getDefaultClearDbServiceConfigMap());
+        assertFalse(cloudFoundryPaasLocation.isServiceBoundTo(SERVICE_INSTANCE_NAME, applicationName));
+        cloudFoundryPaasLocation.bindServiceToApplication(SERVICE_INSTANCE_NAME, applicationName);
```

Comment:
I don't follow, the application and the service are 2 separate app for brooklyn. How can you bind using the `cloudFoundryPaasLocation` ?